pkg/scheduler/placement/placement.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2019 Cloudera, Inc.  All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package placement
+
+import (
+    "fmt"
+    "github.com/cloudera/yunikorn-core/pkg/cache"
+    "github.com/cloudera/yunikorn-core/pkg/common/configs"
+    "github.com/cloudera/yunikorn-core/pkg/log"
+    "go.uber.org/zap"
+    "sync"
+)
+
+type Manager struct {
```

Comment:
yes, will update

Reply:
The IDE complains when the Type name starts with the package name.
Using the package name in the type name is referenced as "Stutter" in Go naming and should be avoided.
Leaving the name as it is.

Reply:
How about `AppPlacementManager`, just `Manager` is too general. When we do searching symbols in IDE, it can easily show up which might confuse developers.

Reply:
yeah that will work